task_2/src/middlewares/verifyJWT.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, NextFunction } from 'express'
+import jwt from 'jsonwebtoken'
+import dotenv from 'dotenv'
+import { AUTHORIZATION_HEADER_NOT_PROVIDED, AUTHORIZATION_TOKEN_INVALID } from '../services/constants.js'
+dotenv.config()
+
+export const verifyJWT = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
it would be great if you rename the method to just `auth` or `authorization` because later you may need change this method for role management as well

Reply:
Thanks for the suggestion, updated